app/policies/authorizer.rb

```diff
@@ -165,10 +165,6 @@ def warned?
       has_role?(:warned)
     end
 
-    def workshop_eligible?
```

Comment:
There is a test in `models/user_spec.rb` which expects this method, let's delete the test.